common/src/main/java/io/seata/common/exception/FrameworkErrorCode.java

```diff
@@ -23,88 +23,99 @@
  */
 public enum FrameworkErrorCode {
     /**
-     * 0001 ~ 0099  与配置相关的错误
+     * 0001 ~ 0099  Configuration related errors
      */
-    ThreadPoolFull("0004", "netty线程池满", "请在配置文件中调整线程数， corePoolSize 的值调大一些"),
+    ThreadPoolFull("0004", "Thread pool is full", "Please check the thread pool configuration"),
+
     /**
      * The Init services client error.
      */
-    InitSeataClientError("0008", "seataAppName or seataServerGroup is null", ""),
+    InitSeataClientError("0008", "Seata app name or seata server group is null", "Please check your configuration"),
+
     /**
      * The Null rule error.
      */
-    NullRuleError("0010", "services rules is null", ""),
+    NullRuleError("0010", "Services rules is null", "Please check your configuration"),
 
     /**
-     * 0101 ~ 0199 网络有关的错误，连接不上，断开，dispatch等
+     * 0101 ~ 0199 Network related error. (Not connected, disconnected, dispatched, etc.)
      */
-    NetConnect("0101", "无法连接服务器", "请检查seata server是否启动，到seata server的网络连接是否正常"),
+    NetConnect("0101", "Can not connect to the server", "Please check if the seata service is started. Is the network connection to the seata server normal?"),
+
     /**
      * The Net reg appname.
      */
-    NetRegAppname("0102", "register client app name failed", "请检查seata server是否启动，到seata server的网络连接是否正常"),
+    NetRegAppname("0102", "Register client app name failed", "Please check if the seata service is started. Is the network connection to the seata server normal?"),
+
     /**
      * The Net disconnect.
      */
-    NetDisconnect("0103", "seataConnection closed", "网络断开，请检查到对端（client 或seata server）的网络连接"),
+    NetDisconnect("0103", "Seata Connection closed", "The network is disconnected. Please check the network connection to the port (client or seata server)."),
+
     /**
      * The Net dispatch.
      */
-    NetDispatch("0104", "dispatch 错误", "网络处理错误，请检查到对端（client 或seata server）的网络连接 "),
+    NetDispatch("0104", "Dispatch error", "Network processing error. Please check the network connection to the port (client or seata server)."),
+
     /**
      * The Net on message.
      */
-    NetOnMessage("0105", "on message 错误", "网络处理错误，请检查到对端（client 或seata server）的网络连接 "),
+    NetOnMessage("0105", "On message error", "Network processing error. Please check the network connection to the port (client or seata server)."),
```

Comment:
port?

Reply:
endpoint ? 👀

Reply:
how about "Network processing errors, check the network connection to the client or Seata server"